export type Colors = {
	systemRed: Color3,
	systemOrange: Color3,
	systemYellow: Color3,
	systemGreen: Color3,
	systemTeal: Color3,
	systemBlue: Color3,
	systemIndigo: Color3,
	systemPurple: Color3,
	systemPink: Color3,
	systemGray: Color3,
	systemGray2: Color3,
	systemGray3: Color3,
	systemGray4: Color3,
	systemGray5: Color3,
	systemGray6: Color3,
	systemBackground: Color3,
	secondarySystemBackground: Color3,
	tertiarySystemBackground: Color3,
	systemGroupedBackground: Color3,
	secondarySystemGroupedBackground: Color3,
	tertiarySystemGroupedBackground: Color3,
	systemFill: Color3,
	secondarySystemFill: Color3,
	tertiarySystemFill: Color3,
	quaternarySystemFill: Color3,
	lightText: Color3,
	darkText: Color3,
	label: Color3,
	secondaryLabel: Color3,
	tertiaryLabel: Color3,
	quaternaryLabel: Color3,
	link: Color3,
	placeholderText: Color3,
	separator: Color3,
	opaqueSeparator: Color3,
}

export type HorizontalAlignment = "Leading" | "Center" | "Trailing"

return nil
